### ISSUE TITLE
Missing prod command

### ISSUE THREAD

**Issue by pedroresende** [filp/ts-package-boilerplate#1]
In the list of the available scripts, the `prod` or `production` execution command is missing

```yaml
  "scripts": {
    "lint": "eslint .",
    "test": "jest .",
    "build": "tsc -b .",
    "dev": "ts-node src/index.ts",
    "prepack": "yarn lint && yarn test && yarn build"
  },
```

**Comment by pedroresende** [comment]
A possible implementation https://github.com/filp/ts-package-boilerplate/pull/2

**Comment by filp** [comment]
Fixed via #2
